tests/fast_tests/test_examples.py

```diff
@@ -155,8 +161,30 @@ def test_density_exp(self):
         run_bottleneck.remote(100, 1, 10, render=False)
 
 
+class TestStableBaselineExamples(unittest.TestCase):
+    """Tests the example scripts in examples/stable_baselines.
+
+        This is done by running each experiment in that folder for five time-steps
+        and confirming that it completes one rollout with two workers.
```

Comment:
```suggestion
     and confirming that it completes one rollout with two workers.
```

Reply:
the suggestions are for pydocstyle compatibility. Technically not needed though since the tests are not checked in the procedure.